addons/im_livechat/static/src/embed/common/livechat_button.js

```diff
@@ -52,16 +52,31 @@ export class LivechatButton extends Component {
             animateNotification: !(
                 this.livechatService.thread || this.livechatService.shouldRestoreSession
             ),
+            hasAlreadyMovedOnce: false,
         });
         useMovable({
             cursor: "grabbing",
             ref: this.ref,
             elements: ".o-livechat-LivechatButton",
             onDrop: ({ top, left }) => {
+                if (!this.state.hasAlreadyMovedOnce) {
```

Comment:
We can remove this condition; it adds unnecessary complexity and nesting. If it's already true, nothing changes and it's fine.

Reply:
Okay! Just seen that I also have 2 commits for some reason, will squash all of that

Reply:
Also, last nitpicking regarding the commit message. It would be much easier to read if the two paragraphs were properly split. Add a period at the end of the first one and split them with a line break.

`Previously, [...] 'o_bottom_fixed_element_move_up'.`

Reply:
Should be good now!